### DISCUSSION TITLE
网上邻居   https://blog.03k.org/friends.html

### DISCUSSION THREAD

**Discussion by kkkgo** [kkkgo/blog.03k.org#15] [blog]
https://blog.03k.org/friends.html

**Comment by mokeyjay** [comment]
Hello😋我换域名啦，麻烦更新一下友链吧~

名称：Mokeyjay's Blog - 超能小紫
域名：https://mok.moe/
头像：https://mok.moe/headimg.webp
口号：资深全沾工程师

**Reply by kkkgo** [reply]
好好好
